herddb-core/src/main/java/herddb/model/commands/InsertStatement.java

```diff
@@ -62,4 +62,9 @@ public boolean isUpsert() {
         return upsert;
     }
 
+    @Override
+    public String toString() {
+        return String.format("{InsertStatement={ keyfunction=%s valuesFunction=%s}}",
```

Comment:
remove '{}' that are wrapping InsertStatement

Reply:
Done.